polkadot/contracts/finance2/logic.rs

```diff
@@ -74,33 +74,26 @@ pub struct Accruer {
 }
 impl Accruer {
     pub fn accrue(self) -> (u128, u64) {
-        let now = if self.now < self.updated_at {
-            self.updated_at
-        } else {
-            self.now
-        };
-        let delta = sub(now as u128, self.updated_at as u128);
-        let standard_matured = self.standard_rate.saturating_mul(delta);
-        let emergency_matured = self.emergency_rate.saturating_mul(delta);
+        if self.now > self.updated_at {
```

Comment:
I'd refactor to:
if self.now <= self.updated_at
  return

[...] rest of the code without indentation